com.openterface.openterfaceQT.yaml

```diff
@@ -0,0 +1,42 @@
+app-id: com.openterface.openterfaceQT
+runtime: org.kde.Platform
+runtime-version: "6.7"
+sdk: org.kde.Sdk
+command: openterfaceQT
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --device=all
+  - --filesystem=/run/udev:ro
+
+modules:
+  - shared-modules/libusb/libusb.json
+
+  - name: openterfaceQT
+    buildsystem: simple
+    sources:
+      - type: git
+        url: https://github.com/TechxArtisanStudio/Openterface_QT.git
+        tag: 0.3.7
+
+    build-options:
+      env:
+        PKG_CONFIG_PATH: /app/lib/pkgconfig:${PKG_CONFIG_PATH}
+        LIBRARY_PATH: /app/lib:${LIBRARY_PATH}
+        LD_LIBRARY_PATH: /app/lib:${LD_LIBRARY_PATH}
+        QT_SELECT: qt6
+    build-commands:
+      - lupdate openterfaceQT.pro
+      - lrelease openterfaceQT.pro
+      - qmake "LIBS+=-L/app/lib -lusb-1.0" "INCLUDEPATH+=/app/include/libusb-1.0"
+      - make -j$FLATPAK_BUILDER_N_JOBS
+      - install -Dm755 openterfaceQT /app/bin/openterfaceQT
+      - test -d config && cp -r config /app/bin/ || echo "No config directory found"
```

Comment:
```suggestion
```

I don't see why this would be necessary. it is either there from you build system or it is not. It's very deterministic.

Reply:
either this is copied or this is not. There is no situation where the `test` would have a different outcome.